numpy/distutils/mingw32ccompiler.py

```diff
@@ -440,6 +440,7 @@ def _build_import_library_x86():
     from numpy.distutils import lib2def
 
     def_name = "python%d%d.def" % tuple(sys.version_info[:2])
+    os.makedirs(os.path.join(sys.prefix, 'libs'), exist_ok=True)
```

Comment:
Is this still needed now? I'd assume not?

Reply:
No.